pallets/schemas/src/rpc/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+use codec::Codec;
+use core::result::Result as CoreResult;
+use jsonrpc_core::{Error as RpcError, ErrorCode, Result};
+use jsonrpc_derive::rpc;
+use schemas_runtime_api::SchemasRuntimeApi;
+use sp_api::{ApiError, ProvideRuntimeApi};
+use sp_blockchain::HeaderBackend;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT, DispatchError};
+use std::sync::Arc;
+
+#[rpc]
+pub trait SchemasApi<BlockHash, AccountId> {
+	#[rpc(name = "schema_get_latest_schema_id")]
```

Comment:
🛑 Blocker:  this should be `schemas_getLatestSchemaId` based on how js library expects it

the convention is
`<pallet name>_<rpc name camel case>`